hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerDataYaml.java

```diff
@@ -250,11 +253,32 @@ public void testIncorrectChecksum() {
       File file = new File(classLoader.getResource(containerFile).getFile());
       KeyValueContainerData kvData = (KeyValueContainerData) ContainerDataYaml
           .readContainerFile(file);
-      ContainerUtils.verifyChecksum(kvData);
+      ContainerUtils.verifyChecksum(kvData, conf);
       fail("testIncorrectChecksum failed");
     } catch (Exception ex) {
       GenericTestUtils.assertExceptionContains("Container checksum error for " +
           "ContainerID:", ex);
     }
   }
+
+  /**
+   * Test to verify disabled checksum with incorrect checksum.
+   */
+  @Test
+  public void testDisabledChecksum(){
+    try {
+      String containerFile = "incorrect.checksum.container";
+      //Get file from resources folder
+      ClassLoader classLoader = getClass().getClassLoader();
+      File file = new File(classLoader.getResource(containerFile).getFile());
+      KeyValueContainerData kvData = (KeyValueContainerData) ContainerDataYaml
+              .readContainerFile(file);
```

Comment:
Can we move this to a common method to get kvData since it is being used by the "testIncorrectChecksum" method as well?